modules/snooker/src/main/ts/ephox/snooker/resize/RuntimeSize.ts

```diff
@@ -1,37 +1,56 @@
 import { HTMLElement } from '@ephox/dom-globals';
 import { PlatformDetection } from '@ephox/sand';
-import { Css, Element, Height } from '@ephox/sugar';
+import { Css, Element, Height, Width } from '@ephox/sugar';
 
-const needManualCalc = function () {
-  const platform = PlatformDetection.detect();
-  return platform.browser.isIE() || platform.browser.isEdge();
+const needManualCalc = () => {
+  const browser = PlatformDetection.detect().browser;
+  return browser.isIE() || browser.isEdge();
```

Comment:
How does `isEdge` work with new edge vs old edge? Also is it feasible to check this by looking at behaviour instead of checking browser names?

Reply:
Unrelated to this PR.